var/ramble/repos/builtin/applications/py-nemo/application.py

```diff
@@ -27,6 +27,12 @@ class PyNemo(ExecutableApplication):
 
     tags("ml-framework", "machine-learning")
 
+    executable(
+        "setup_transformer_cache",
+        'bash -c "python3 -c \'from transformers import AutoTokenizer; AutoTokenizer.from_pretrained(\\"gpt2\\")\'"',
```

Comment:
I think so, but I am not the expert on this.

I think all this does is allow the transformers module to setup a cache that it will store transformers in.